### DISCUSSION TITLE
What's the policy for iFrame / embedding?

### DISCUSSION THREAD

**Discussion by getlarge** [ClaperCo/Claper#89] [General]
Hey, I just discovered Claper and would like to embed forms, polls, and other such features in some Reveal.js presentations (more precisely, in slides.com).

Unfortunately, the event presentation (`https://app.claper.co/e/event_id/presenter`) cannot be embedded due to the X-FRAME-OPTIONS header, which is probably set to SAMEORIGIN or DENY (surprisingly, the attendee room can be embedded).
When I looked at the code to search where this header could be set, I only found [this](https://github.com/ClaperCo/Claper/blob/6d26c0ec86a18021d092df4f214f4023bb152507/lib/claper_web/plugs/iframe.ex#L3) which would allow iFrame, and I am now confused!

Could anyone share an explanation?

**Comment by alxlion** [comment]
Hello,

Iframe embed is only possible for non-user routes like `/` and  `/e/[event_code]`. It is not possible to embed event manager at the moment.

**Reply by getlarge** [reply]
Thanks for the explanation
